cocotb/decorators.py

```diff
@@ -374,7 +374,7 @@ def _wrapped_test(*args, **kwargs):
             try:
                 return RunningTest(self._func(*args, **kwargs), self)
             except Exception as e:
-                raise raise_error(self, str(e))
+                raise raise_error(self, "test raised error")
```

Comment:
To stay consistent with the other cases you might want to make this `raise_error(self, "Test raised error:")`

Reply:
Done in #716